core/src/main/kotlin/api/OrganizationsRoute.kt

```diff
@@ -238,9 +239,8 @@ fun Route.organizations() = route("organizations") {
                         createSecret.name,
                         createSecret.value,
                         createSecret.description,
-                        organizationId,
-                        null,
-                        null
+                        Entity.ORGANIZATION,
+                        organizationId
```

Comment:
For most callers it would be nicer to have convenience functions like `createRepositorySecret` or `createProductSecret` (probably with default implementations in the interface as implementations would be identical).

Reply:
Where's the convenient in using these functions that basically just encode the entity parameter as part of the function name? They are hardly shorter, and create a maintenance burden if more entities (like business units, groups of organizations etc.) would be added in the future.

Reply:
I guess it's a matter of taste as well, personally I'd rather call `getOrganizationSecrets(orgId)` than `getSecrets(Entity.ORGANIZATION, orgId)` because it has less parameters.

Reply:
So, not a *must* to address? Because I'd prefer not to.

Reply:
No, not a must, but as this is still a draft maybe there is some time to get an opinion from another contributor (@oheger-bosch @MarcelBochtler @bs-ondem)?